docs/introduction/blockchain-representation.rst

```diff
@@ -2,51 +2,51 @@
 
 Concept of a BOX
 ****************
 
 A box generalizes the concept of Bitcoin’s UTXOs.
-A box is a cryptographic object that can be created with some secret keys. This box can be open \ (spent) by the owner of those secret keys. Once opened by the owner of the secret keys the box may not be opened again.
+A box is a cryptographic object that can be created with some (one private key or multiple private keys?) secret keys . This box can be opened (spent) by the owner(owners ?) of those secret keys. Once opened by the owner (owners ?) of the secret keys the box may not be opened again.
 
 Node Main elements & intro to a "NodeView"
 ******************************************
 
   * **History**
-    * “History” is a blockchain ledger, that is typically a list of Sidechain blocks that were received by the Node, and that have been verified against Consensus rules, and accepted.
+    * “History” is a blockchain ledger, typically a list of sidechain blocks that were received by the node, have been verified against Consensus rules, and accepted.
     
   * **State**
     * “State” is a snapshot of all boxes that haven’t been opened yet. It represents the state at the current chain tip.
     
   * **Wallet**
     * The “Wallet” has two main functionalities:
-      1. It holds the Secret keys that belong to that specific Node.
+      1. It holds the secret keys that belong to that specific node.
       2. It keeps track of objects that are of interest to this specific node, e.g. received coins (output boxes whose secret keys are known by the node) and views of them (e.g. balances).      
   * **Memory Pool**
-    * The “Memory pool” is a list of transactions that are known to the node but have not made it to a Sidechain block yet.
+    * The “Memory pool” is a list of transactions that is known to the node but have not made it to a Sidechain block yet.
     
 Altogether these 4 objects represent a “NodeView.”
 
 NodeViewHelper
 ==============
 
-All communication between NodeView objects is controlled by NodeViewHolder, which also provides a layer of communication within the application for local data processing of Blocks, Transactions, Secrets, etc.
+All communication between NodeView objects is controlled by NodeViewHolder, which also provides a layer of communication within the application for local data processing of blocks, transactions, secrets, etc.
 
 In terms of customization, the History object is the only one that is fully controlled by the core and that in almost all circumstances does not need to be extended. It contains a ready-made implementation of the Latus consensus and of the Cross-Chain Transfer Protocol.
 
-The core logic of State, Wallet and Memory Pool can instead be extended by sidechain developers:
+The core logic of State, Wallet and Memory Pool objects can be extended by sidechain developers:
```

Comment:
@zennermac 

The core logic of state, wallet, and memory pool objects can be extended by sidechain developers: